ViewModel/AuctionItemCollectionRenderer.php

```diff
@@ -42,6 +42,6 @@ public function __construct(
      */
     public function getAllAuctionItems()
     {
-        return $this->auctionItemRepository->getList($this->searchCriteriaBuilder->create());
+        return $this->auctionItemRepository->getList($this->searchCriteriaBuilder->create())->getItems();
```

Comment:
Now that the return type is different, the docblock should be updated too.

I also like adding type declarations for return types. But you need to be careful because these are enforced at runtime. If you declare an `int` return type and return `null` it'll throw a `TypeError`.
I think it's best to provide as much information as possible when it comes to types, it makes IDE auto-completion work better and improves the experience for future you and other developers. 

You could update this to something like:
```
/**
     * Returns collection of all Contacts
     *
     * @return AuctionItem[]
     */
    public function getAllAuctionItems(): array
    {
        return $this->auctionItemRepository->getList($this->searchCriteriaBuilder->create())->getItems();
    }
```
In docblocks you can specify the type of the values in an array, but not for argument/return type declarations since this can't be enforced at runtime.